lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
btw: in my Zsh config I get three completions for `tru` (external command, builtin command, local directory), but only because it could also be `truncate` or `trust`.
`true` completes to just `true` as a command then.
If there is also `truedir` I get completions for `true` (external command, builtin) and `true/`, `truedir/`.